tests/unit/test-data/registration.json

```diff
@@ -40,7 +40,7 @@
       },
       "businessType": "SP",
       "contactPoint": {
-        "email": "eleven@email.com",
+        "email": "eleven@example.com",
```

Comment:
This is a non-routable email address, whereas `email.com` is a legitimate domain.